frontend/app/controllers/auth_controller.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+module.exports = function(app) {
+  app.controller('AuthController', ['$http', '$location', function($http, $location) {
+
+    this.user = {};
```

Comment:
Okay, line 6 was old code I'd removed in the branches following this one. My plan was to make that object accessible by other components but it makes more sense to throw it in a service.